websites/mswjs.io/src/content/docs/recipes/custom-worker-script-location.mdx

```diff
@@ -12,6 +12,10 @@ await worker.start({
     // This is useful if your application follows
     // a strict directory structure.
     url: '/assets/mockServiceWorker.js',
+    options: {
+      // Required if you opt-in to Service-Worker-Allowed header
+      scope: "YOUR_TARGET_SCOPE",
```

Comment:
Oh, so the `scope` option becomes required if you use a custom `Service-Worker-Allowed` header? I didn't know that. 

@cdaringe could you please post what kind of error were you having before this fix? I'd love to learn more.

Reply:
Got it! Would appreciate if you posted the original warning though. Makes it easier for people to discover this in hindsight. Thanks.